### DISCUSSION TITLE
is sending JPG attachment fully supported ?

### DISCUSSION THREAD

**Discussion by 16SR** [tutao/tutanota#3315] [Q&A]
I'm frequently sending JPG files below 20 mb. Are these fully supported ?
Reason I'm asking, one of my customers who ordered a poster as a digital file, reported that the file she received was in the wrong format and she couldn't print it and the printer she took the file to also couldn't print.  She didn't explain what she means. After explaining that the file could have been damaged during saving it either to computer or to USB memory stick.
Problem solved in the end, but I'm curious if Tutanota handles JPG files 100% ?

**Comment by johnbotris** [comment]
Tutanota handles attachments in a format agnostic way. You should make sure that the `.jpg` extension is in the filename, and that the customer isn't removing it when she saves the file, as windows relies on the file extension to know how to interpret a file.

**Reply by 16SR** [reply]
Thanks!
Yes I know that some people may save JPG file without the ending .jpg
Support for HTML is not 100%.  For example you can't use colors for words fonts/words. You can't select different fonts either. BUT if I copy>paste text with a lot of colored  words from desktop publishing application for example, it is pasted ok, everything preserved.
